### PR TITLE
An Attempt at recording Unstable procs

### PR DESCRIPTION
**Pull request by kuhmann**
this is super modelled after cauldrons with just some minor edits.

### CODE REVIEW

**Comment by DevJackSmith** [comment]
@kuhmann not sure if you saw but there's a lint thing that wants you to add a comma

**Comment by DevJackSmith** [comment]
also was this tested? (you can load the code as unpacked extension, then open devtools, and see what it tries to send on the network tab to intake.php)

File: src/scripts/main.js [hunk 1]
```diff
@@ -990,6 +990,29 @@
...
+                            id: 1478, // Unstable Charm's item ID
+                            name: "Unstable Charm",
+                            quantity: 1,
+                        };
+                        const items = [{
+                            id: trinketId,
+                            name: trinketName,
+                            quantity: 1
+                        }];
+                        if (debug_logging) { window.console.log({message:"Submitting Unstable Charm Data to MHCT: ", unstable_charm_loot: items}); }
```
Comment (DevJackSmith) [inline src/scripts/main.js:1010]:
```suggestion
                        if (debug_logging) { window.console.log({message:"MHCT: Submitting Unstable Charm: ", unstable_charm_loot: items}); }
```

File: src/scripts/main.js [hunk 2]
```diff
@@ -990,6 +990,29 @@
...
+                        const {name: trinketName, item_id: trinketId} = hunt_response.inventory[resultTrinket];
+                        const convertible = {
+                            id: 1478, // Unstable Charm's item ID
+                            name: "Unstable Charm",
+                            quantity: 1,
+                        };
+                        const items = [{
+                            id: trinketId,
+                            name: trinketName,
+                            quantity: 1
```
Comment (DevJackSmith) [inline src/scripts/main.js:1008]:
```suggestion
                            quantity: 1,
```

**Review by AardWolf — APPROVED** [review]
Closely follows existing examples, compiles, should work!

**Review by DevJackSmith — APPROVED** [review]

**Pull request merged**